04-rstudio-tour.Rmd

```diff
@@ -1,38 +1,277 @@
+   - Check if maybe you need a different version for the age of your system. First check that your version of R was right - there are multiple versions for different `macOS` systems for example. You can check the apple icon (top left corner) and "About This Mac" to learn more about the age of your operating system. 
+   
+If your operating system is older (and you can't update it), try installing progressively older versions found [here](https://www.rstudio.com/products/rstudio/older-versions/) until it works. You will know if it worked if you try to open RStudio and you see an interface without a message about things going poorly. Here you can see an [example](https://community.rstudio.com/t/rstudio-desktop-crashes-on-startup-with-library-not-loaded/130296) of this. 
+
+</details>
+
+### Updating RStudio
+
+It is also a good idea to keep RStudio up-to-date. New features become available as the Posit team works on developing RStudio. So if you already have RStudio, you might want to check to see if your version is up-to-date.
+
+To check for updates you can go to the `Help` menu at the top of RStudio and then click on `Check for Updates`. 
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Check for updates for RStudio in the Help menu of RStudio."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_50")
+```
+
+## Navigating RStudio
+
+Now that you hopefully have RStudio running on your machine, we will walk you through some of the major features that can really help you with your data analyses.
+
+### Default Layout
+
+First it is important to be familiar with the layout. When you first open RStudio you will see 3 panes.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "When RStudio is first opened you will see 3 panes"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_67")
+```
+
+<details><summary>If your RStudio looks different click here.</summary>
+
+Click on the top menu of your RStudio - click where it says `Edit` --> `Preferences` --> `Pane Layout`.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Select Edit and then Preferences to get to the menu about pane layout."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g22370a5f292_430_0")
+```
+
+The Pane Layout menu enables you to change the layout. The image below shows the default settings. 
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The Pane Layout menu allows you to modify the layout of RStudio. This shows the default settings with Source on the top left, Console on the bottom left, Environment, History, Connections, and Build on the top right, and Files, Plots, Packages, Help, and VCS on the bottom right."}
```

Comment:
What's VCS?

Reply:
a typo lol

Reply:
oh maybe I ment version control or something

Reply:
oh not a typo it is in the menu

Reply:
I don't have VCS in my RStudio menu. Should we address that?